src/app/components/champion/ChampionSelector.tsx

```diff
@@ -1,43 +1,44 @@
-import React, {useCallback} from "react";
-import { useEffect, useState } from "react";
-import { Champion, Data } from "../model/models";
-import { ChampionView } from "./ChampionView";
+import React, {useCallback, useEffect, useState} from "react";
+import {Champion, Data} from "../model/models";
+import {ChampionView} from "./ChampionView";
+import {getUrlChampion, getUrlChampionList} from "../../config/config";
+import axios from 'axios';
+import {defaultChampion} from "./default";
 
 
 export const ChampionSelector = () => {
-    const [champion, setChampion] = useState<Champion>({name: 'Aatrox', title: 'the Destroyer', blurb : 'Placeholder'},);
-
-    const championsList = [
-        {name: 'Aatrox', title: 'the Destroyer', blurb : 'Placeholder'},
-        {name: 'Gangplank', title: 'the Pirate', blurb : 'Placeholder'},
-        {name: 'Yorick', title: 'the Grave digger', blurb : 'Placeholder'},
-        {name: 'Ahri', title: 'the Cat', blurb : 'Placeholder'},
-    ]
+    const [champion, setChampion] = useState<Champion>(defaultChampion);
+    const [championsList, setChampionsList] = useState<Champion[]>([]);
 
     useEffect(() => {
-        loadChampion();
+        if (championsList.length == 0) fetchChampionList()
+        loadChampion(champion.id);
     }, [])
 
-    function loadChampion() {
-        fetch(`http://ddragon.leagueoflegends.com/cdn/11.6.1/data/en_US/champion/Aatrox.json`, {
-            method: 'GET',
-        }).then(response => response.json())
-            .then(response => {
-                const payload  = response as Data
-                const champs = Object.values(payload.data) as Champion[]
-                const champion = champs[0]
-                setChampion(champion as Champion)
-            })
+    function fetchChampionList() {
+        axios.get(getUrlChampionList(), {}).then(response => {
+            const payload = response.data as Data
+            const champs = Object.values(payload.data) as Champion[]
+            setChampionsList(champs)
+        })
+    }
+
+    function loadChampion(championSelected:string) {
+        axios.get(getUrlChampion(championSelected), {}).then(response => {
+            const payload = response.data as Data
+            const champs = Object.values(payload.data) as Champion[]
```

Comment:
Use lodash ? Look if it fits to your needs

Reply:
not mandatory for this PR.

Reply:
I looked a the documentation, it may be useful a little later when manipulating stats but I won't need it for now.